src/SFA.DAS.AssessorService.Api.Types/Models/SearchAssessmentOrganisationsRequest.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+using MediatR;
+using SFA.DAS.AssessorService.Api.Types.Models.AO;
+
+namespace SFA.DAS.AssessorService.Api.Types.Models
+{
+    public class SearchAssessmentOrganisationsRequest : IRequest<List<AssessmentOrganisationSummary>>
+    {
+        public string Searchstring { get; set; }
```

Comment:
Just wondering about the naming of this ? almost like hungarian notation - I would remove string on SearchString.

Reply:
I just meant it as a usage of the expression 'search string', and I had precedence from other parts of the codebase using it the same way (not that this is a good reason?).  I personally think we do need to guard against adding type to variables, but 'string' here isn't there for indicating type, but meaning.